src/package.xml

```diff
@@ -944,9 +964,16 @@
         <members>Trigger_Handler__c.All_Main_Fields</members>
         <name>ListView</name>
     </types>
+    <types>
+        <members>Admin</members>
+        <name>Profile</name>
+    </types>
     <types>
         <members>Attribute__c.Credential</members>
         <members>Attribute__c.Student_Characteristic</members>
+        <members>Course_Enrollment__c.Default</members>
```

Comment:
@kdy618 I think you mistakenly added these record types to the package.xml. We deploy these as unmanaged, so they shouldn't be included in the package.

Reply:
@robertbwatson that's really odd. It might be when I initially tried to commit, my child branch was behind and I pulled down first and recommit my changes. I should remove Course_Enrollment__c.Default right? I noticed that Course_Enrollment__c.Faculty & Student was also added. Should I remove those Course_Enrollment__c record types?

Reply:
Yes - all three of these record types should be removed. We don't want them in the package - they were released a year or two ago as unmanaged and must not be added.

Reply:
There are a few interesting bits to our codebase ;) We have to make sure that the stuff in the /unpackaged directory doesn't get added to the package - only what we have in the /src directory of the repo.